searx/templates/oscar/base.html

```diff
@@ -85,10 +85,10 @@
             {% endblock %}
             <p class="text-muted">
                 <small>
-                    {{ _('Powered by') }} <a href="https://asciimoo.github.io/searx/">searx</a> - {{ searx_version }} - {{ _('a privacy-respecting, hackable metasearch engine') }}<br/>
-                    <a href="https://github.com/asciimoo/searx">{{ _('Source code') }}</a> |
-                    <a href="https://github.com/asciimoo/searx/issues">{{ _('Issue tracker') }}</a> |
-                    <a href="https://searx.space/">{{ _('Public instances') }}</a>
+                    {{ _('Powered by') }} <a href="{{ brand.DOCS_URL }}">searx</a> - {{ searx_version }} - {{ _('a privacy-respecting, hackable metasearch engine') }}<br/>
+                    <a href="{{ brand.GIT_URL }}">{{ _('Source code') }}</a> |
+                    <a href="{{ brand.GIT_URL }}/issues">{{ _('Issue tracker') }}</a> |
```

Comment:
It supposes that the URL will always follow the same schema, even if github is not hosting the repository.

I suggest to add an ISSUE_URL variable. Perhaps far-fetched.

Reply:
As longer I think about I came to the conclusion that the issue tracker URL should be hard coded in the template.

Reply:
I mean .. if it is a brand, the issue tracker should be the same

Reply:
Example:
* GIT_URL=https://framagit.org/framasoft/searx
* ISSUE_URL=https://framagit.org/framasoft/searx/-/issues 

There is an additional ```-/``` in the URL, most probably for any gitlab repository.

Reply:
OK .. done